src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/AzureUtils.java

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
Updated. 

And I think we should have some talk about the license. There are several related issues.